src/app/pages/TopicPage/TopicPromo/index.jsx

```diff
@@ -8,13 +8,14 @@ const TopicPromo = ({
   firstPublished,
   imageUrl,
   imageAlt,
+  lazy,
   link,
   mediaType,
   mediaDuration,
 }) => {
   return (
     <Promo>
-      <Promo.Image src={imageUrl} alt={imageAlt}>
+      <Promo.Image src={imageUrl} alt={imageAlt} loading={lazy ? 'lazy' : null}>
```

Comment:
Maybe "eager" instead of null

Reply:
> Maybe "eager" instead of null

Actually, found this. 

"The default is eager, so it doesn’t need to be explicitly added (that is, just use <img> for eager loading)."

So, as you were, I'll get my coat etc